### PR TITLE
fix: LSDV-3854-1: Add extension check

### PR DESCRIPTION
**Pull request by triklozoid**

### CODE REVIEW

File: label_studio/core/settings/base.py [hunk 1]
```diff
@@ -400,6 +400,31 @@
 UPLOAD_DIR = 'upload'
 AVATAR_PATH = 'avatars'
 
+SUPPORTED_EXTENSIONS = set(
```
Comment (farioas) [inline label_studio/core/settings/base.py:403]: There are no video formats in this list.
Here's Chrome supported formats: https://sites.google.com/a/chromium.org/dev/audio-video

Comment (triklozoid) [reply label_studio/core/settings/base.py:403]: Added video extensions

**Review by makseq — CHANGES_REQUESTED** [review]
Let's add ff

File: label_studio/data_import/uploader.py [hunk 2]
```diff
@@ -58,6 +58,12 @@ def check_tasks_max_file_size(value):
         raise ValidationError(f'Maximum total size of all files is {settings.TASKS_MAX_FILE_SIZE} bytes, '
                               f'current size is {total} bytes')
 
+def check_extensions(files):
+    for filename, file_obj in files.items():
+        _, ext = os.path.splitext(file_obj.name)
```
Comment (makseq) [inline label_studio/data_import/uploader.py:63]: If file doesnt have extension, it fails.

File: label_studio/tests/data_import.tavern.yml [hunk 3]
```diff
@@ -733,3 +733,39 @@ stages:
...
+    files:
+      file: tests/test_suites/samples/test.wrong_ext
+    headers:
+      content-type: multipart/form-data
+    method: POST
+    url: '{django_live_url}/api/projects/{project_pk}/import?commit_to_project=false'
+  response:
+    json:
+      validation_errors:
+        non_field_errors:
```
Comment (makseq) [inline label_studio/tests/data_import.tavern.yml:768]: Let's add file without extension in test too

**Review by makseq — CHANGES_REQUESTED** [review]

File: label_studio/tests/data_import.tavern.yml (hunk 3 above)
Comment (triklozoid) [reply label_studio/tests/data_import.tavern.yml:768]: Added.

File: label_studio/data_import/uploader.py (hunk 2 above)
Comment (triklozoid) [reply label_studio/data_import/uploader.py:63]: Can you provide an example? 
For me is works ok: 
```
In [1]: import os                                                                         

In [2]: filename = 'test'                                                                 

In [3]: os.path.splitext(filename)                                                        
Out[3]: ('test', '')
```

**Review by makseq — APPROVED** [review]

**Pull request merged**